main.go

```diff
@@ -21,13 +21,13 @@ func main() {
 	err := godotenv.Load(*envFile)
 	if err != nil {
 		log.Fatalf("Error loading .env file at %s. Err: %s\n", *envFile, err)
+		return
 	}
 
 	debug, err := strconv.ParseBool(os.Getenv("DEBUG"))
 	if err != nil {
-		debug = true // default
+		debug = false // default
```

Comment:
This only controls value of debug when invalid value is specified in `.env` file. Are you trying to make default value for debug as false? For that, you should update it in `.env.sample`. I think, this value here, is better off with true, since .env value was messed up and then its probably better to show more value, kind of like disabling it when its explicitly specified as false, otherwise be more verbose.

Reply:
Here's what happened:

I was deploying neurobot to production, and initially just copy-pasted the values from `.env.sample`, in which `DEBUG` is set to `true`. However, in a production environment, I wanted debug disabled, so I removed the `DEBUG=true` line from my `.env`.

Then I started `neurobot` and it logged `Debug: true`. So I'm thinking, "I didn't set DEBUG to true, so neurobot must not be finding my .env, and is instead using some internal defaults?". So I go look into the source, and then realise that when the `DEBUG` variable isn't defined, it defaults to true.

In my view this is unexpected. If I don't set DEBUG to true, I expect it to be false.

In any case, I think it makes sense to also revisit this when[ looking at logging in general](https://github.com/Automattic/neurobot/issues/1), so I'll undo this change.

Reply:
I agree with default value of `debug` being false, but I didn't think of handling cases where someone would just remove the config line from config, or comment it out, which is pretty common in linux too.

So, I am going to flip debug=false as default unless specified true in `.env`

Reply:
Issue created - https://github.com/Automattic/neurobot/issues/29